### DISCUSSION TITLE
Getting Error: Provided user was not an "AzureUser". while getting container from Azure Fluid Relay Service

### DISCUSSION THREAD

**Discussion by rohitvpatil0810** [microsoft/FluidFramework#19411] [Q&A] [answered]
While some client connections work fine, After some no. of clients are connected when the next client tries to connect to the same container I get this error.
`Error: Provided user was not an "AzureUser".`

Here is my connectionConfig
```
export const serviceConfig: AzureClientProps = {
  connection: {
    tenantId: "", // REPLACE WITH YOUR TENANT ID
    tokenProvider: new InsecureTokenProvider(
      "" /* REPLACE WITH YOUR PRIMARY KEY */,
      user
    ),
    endpoint: "", // REPLACE WITH YOUR SERVICE ENDPOINT
    type: "remote",
  },
};

const connectionConfig:
  | AzureRemoteConnectionConfig
  | AzureLocalConnectionConfig = useAzure
  ? remoteConnectionConfig
  : localConnectionConfig;
export const clientProps: AzureClientProps = {
  connection: serviceConfig.connection,
  logger: devtoolsLogger,
};
```
I am using InsecureTokenProvider, what is the significance of the `user` when testing in a local environment using Azure Fluid Relay Service?

```
const client = new AzureClient(clientProps);
({ container, services } = await client.getContainer(
      containerId,
      containerSchema
    ));
```

I get this error on getContainer Method.

**Comment by nmsimons** [comment]
I think I figured out the issue. The user in these examples is fake (obviously) and to make the refresh experience "better", the sample used local storage to track the fake user id. However, this seems to cause the issue you encountered. We need to investigate precisely what is happening, but this branch has a fix: https://github.com/microsoft/FluidExamples/tree/RemoveUserFromLocal. Note, that there is a PR open to merge that branch and once it is merged, the main branch will have the fix. But if you happen to read this immediately, you can try the branch.

**Reply by nmsimons** [reply, marked as answer]
The fix is now merged into main.
